src/main/java/com/javadiscord/javabot/commands/reaction_roles/HelpRole.java

```diff
@@ -0,0 +1,62 @@
+package com.javadiscord.javabot.commands.reaction_roles;
+
+import net.dv8tion.jda.api.entities.ChannelType;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.events.message.react.MessageReactionAddEvent;
+import net.dv8tion.jda.api.events.message.react.MessageReactionRemoveEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+public class HelpRole  extends ListenerAdapter {
```

Comment:
This class is missing javadoc. Please add a description of what this listener does. Also remove the extra space between the class name and `extends`.